build/devenv/services/tokenVerifier.go

```diff
@@ -0,0 +1,255 @@
+package services
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/Masterminds/semver/v3"
+
+	chainsel "github.com/smartcontractkit/chain-selectors"
+	"github.com/smartcontractkit/chainlink-ccip/ccv/chains/evm/deployment/v1_7_0/operations/executor"
+	onrampoperations "github.com/smartcontractkit/chainlink-ccip/ccv/chains/evm/deployment/v1_7_0/operations/onramp"
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/deployment/v1_6_0/operations/rmn_remote"
+	aggregator "github.com/smartcontractkit/chainlink-ccv/aggregator/pkg"
+	"github.com/smartcontractkit/chainlink-ccv/devenv/internal/util"
+	"github.com/smartcontractkit/chainlink-ccv/protocol"
+	"github.com/smartcontractkit/chainlink-ccv/verifier/token"
+	"github.com/smartcontractkit/chainlink-deployments-framework/datastore"
+	"github.com/smartcontractkit/chainlink-testing-framework/framework/components/blockchain"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/go-connections/nat"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/framework"
+)
+
+//go:embed tokenVerifier.template.toml
+var tokenVerifierConfigTemplate string
+
+type TokenVerifierInput struct {
+	Mode           Mode                 `toml:"mode"`
+	Out            *TokenVerifierOutput `toml:"-"`
+	Image          string               `toml:"image"`
+	SourceCodePath string               `toml:"source_code_path"`
+	RootPath       string               `toml:"root_path"`
+	ContainerName  string               `toml:"container_name"`
+	Port           int                  `toml:"port"`
+
+	// Maps to on_ramp_addresses in the verifier config toml.
+	OnRampAddresses map[string]string `toml:"on_ramp_addresses"`
+	// Maps to committee_verifier_addresses in the verifier config toml.
+	DefaultExecutorOnRampAddresses map[string]string `toml:"default_executor_on_ramp_addresses"`
+	// Maps to rmn_remote_addresses in the verifier config toml.
+	RMNRemoteAddresses map[string]string `toml:"rmn_remote_addresses"`
```

Comment:
Does the offchain token verifier need these? Or does it need the onchain verifier contracts, e.g USDCVerifier